src/main/java/xyz/vegaone/easytracking/repo/UserRepo.java

```diff
@@ -0,0 +1,10 @@
+package xyz.vegaone.easytracking.repo;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import xyz.vegaone.easytracking.domain.UserEntity;
+
+@Repository
+public interface UserRepo extends JpaRepository<UserEntity, Long>{
```

Comment:
As a general note: do you use CTRL+ALT+L to format code in java classes.

Reply:
nope